runtime_scan/pkg/provider/cloudinit/cloud-init.tmpl.yaml

```diff
@@ -17,20 +17,20 @@ write_files:
       
       [Service]
       Type=oneshot
-      WorkingDirectory=/opt/vmclarity
-      ExecStartPre=mkdir -p /var/opt/vmclarity
+      WorkingDirectory={{ .WorkingDir }}
```

Comment:
It is related to this PR since the constant /opt/vmclarity is not good for google container optimized image, since this part of the disk is read only

Reply:
linux/ubuntu images on gcp does not have docker installed by default, The recommended image for gcp if you want to run containers is [Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs)
This has docker and cloud init installed

Reply:
Actually the ubuntu image that I checked (ubuntu-2004-lts) does not have cloud-init installed.
We can use startup scripts to install it, but not sure this is the approach we want

Reply:
> It is related to this PR since the constant /opt/vmclarity is not good for google container optimized image, since this part of the disk is read only

I understand that, but this still could be a separate change merged prior this PR.

Reply:
Removed this for now until we decide if need this